frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -73,48 +79,57 @@ export default class FirebaseInteractor {
     this.db.collection("users").doc(userAuth.user.uid).set({
       name,
       accountType,
-      age
+      age,
     });
     userAuth.user.sendEmailVerification();
   }
 
-  async signInWithUsernameAndPassword(username : string, password : string) {
+  async signInWithUsernameAndPassword(username: string, password: string) {
     this.unsubscribe?.apply(this);
     await this.auth.signInWithEmailAndPassword(username, password);
     let id = this.auth.currentUser?.uid;
     let user = await this.db.collection("users").doc(id).get();
     let userData = user.data();
     if (id != null && userData != null) {
-      this.currentUser = { id: id, name: userData.name as string, accountType: userData.accountType as AccountType, age: userData.age as number };
+      this.currentUser = {
+        id: id,
+        name: userData.name as string,
+        accountType: userData.accountType as AccountType,
+        age: userData.age as number,
+      };
     }
-    
   }
 
   /**
    * Gets all possible words.
    *
    * @returns { Promise<Array<Word>> }
    */
-  async getWords() : Promise<Word[]> {
-    let wordRef = await this.db.collection("words").get();
+  async getWords(): Promise<Assessment> {
```

Comment:
Can we name this something else? It seems weird to call it getWords and return an assessment